e2e-jar-test/pom.xml

```diff
@@ -23,19 +23,19 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>2.14.0</version>
+        <version>2.17.2</version>
       </dependency>
 
       <dependency>
         <groupId>net.snowflake</groupId>
         <artifactId>snowflake-ingest-sdk</artifactId>
-        <version>2.1.2</version>
+        <version>2.2.0-SNAPSHOT</version>
```

Comment:
why do we need a minor version upgrade?

Reply:
there is no behavior change for these PRs merging into main, I asked them to have a feature branch which will contain all the user interface changes and behavior changes

Reply:
Correct, 2.1.3 should work.
Do we have tooling that diffs the public API surface of our binary between releases? If not automated, we manually compare the public API surface between what's already released and what's being released now?